src/Collector/Console/CommandCollector.php

```diff
@@ -68,13 +68,14 @@ public function collect(ConsoleEvent|ConsoleErrorEvent|ConsoleTerminateEvent $ev
             return;
         }
 
+        $definition = $command?->getDefinition();
         $this->commands[$event::class] = [
-            'name' => $command->getName(),
+            'name' => $command?->getName() ?? $event->getInput()->getFirstArgument() ?? '',
```

Comment:
Are you sure that first argument is a command name?

Reply:
I don't know. I spied it here:

![image](https://github.com/yiisoft/yii-debug/assets/525501/d9e6de77-8c12-4a9c-9199-47bc2398fc8b)